content/docs/command-reference/remote/index.md

```diff
@@ -26,49 +26,34 @@ positional arguments:
 
 ## Description
 
-What is data remote?
+DVC remotes are distributed storage locations for your data sets and ML models
+(similar to Git remotes, but for <abbr>cached</abbr> assets). This optional
+feature is typically used to share or back up copies of all or some of your
+data. Several types are supported: Amazon S3, Google Drive, SSH, HTTP, local
+file systems, among others.
```

Comment:
Also rewrote (shortened) the `remote` ref index page.

Reply:
Looks good! Should we have a link here to the different storage types?

Reply:
Linking [among others]